.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,10 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
 contact_links:
   - name: Usage question
-    url: https://stackoverflow.com/questions/tagged/python-xarray
-    about: "Post a question on Stack Overflow using the #python-xarray
-    tag. These are regularly reviewed by xarray's maintainers, and questions which
-    include a reproducible example will receive a response."
\ No newline at end of file
+    url: https://github.com/pydata/xarray/discussions
+    about: |
+      Ask questions and discuss with other community members here.
+      If you have a question like `How do I concatenate a list of datasets?` then 
+      please include a self-contained reproducible example if possible.
```

Comment:
@max-sixty, let me know if this enough and/or whether this is what you are suggesting in 

> Should we specify the reproducible examples here, or is there a separate template for discussions?

Reply:
Yes great! I'm not sure if we can fit the link we generally use <https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports>, so probably we go with this and add the link when we can have a standard discussion template.

Thanks!